File: SoftuniLabPractice/SoftuniLab/src/softuni_objectsandclasses_folder/Song.java
package softuni_objectsandclasses_folder;

public class Song {
	private String type;
	private String name;
	private String duration;
	
	public Song(String n, String m, String k ) {
		this.type = n;
		this.name = m;
		this.duration = k;
	}
	
	public String getName() {
		return this.name;
	}
	
	public String getType() {
		return this.type;
	}

}

File: SoftuniLabPractice/VisualStudioWorkspace/SoftuniFundamentals/SoftuniFundamentals/Song.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SoftuniFundamentals
{
    class Song
    {
        private string type {  get; set; }
        private string Name
        {
            public get => this.Name;
            private set;
        }
        private string duration {  get; set; }

        public Song()
        {

        }

        public Song(string type, string name, string duration)
        {
            this.type = type;
            this.Name = name;
            this.duration = duration;
        }

       
        
       
    }
}
